mef-sog-uc1

ainnocenti2011 opened the pull request:
fix keda resources nvidia.com/gpu and ephemeral-storage
  fix path kafdrop.ingress.yaml

ktatarnikovhiro commented on integration/apps/vault.yaml:
```diff
@@ -0,0 +1,231 @@
+      - name: minio-kes-service-account-token
+        secret:
+          defaultMode: 420
+          secretName: minio-kes-secret
```
it seems there is circular dependency between minio-kes-secret and minio, or at least there is a dependency between PRs. Could you verify that please?

matthewrossi replied:
You are right, this PRs as it is right now depends on the creation of a secret that is currently part of https://github.com/glaciation-heu/gitops-deployments/pull/45. I completely missed it when I was breaking down the changes in multiple PRs.
By moving `minio-kes-sa-and-secrets.yaml` in this PR, commit https://github.com/glaciation-heu/gitops-deployments/pull/44/commits/df7fb9929e2826ecba5510cc0e75842d29b3f8fb should fix the problem.

matthewrossi commented:
I noticed that Vault was still relying on the values stored in https://github.com/unibg-seclab/glaciation-gitops-test.
To avoid that changes in our test repository may propagate into the integration cluster I have changed the definition of the Vault application to embed its values.

luximu85 committed:
mef-sog-uc1:
  fix keda resources nvidia.com/gpu and ephemeral-storage
  fix path kafdrop.ingress.yaml
